src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/updateSubmission/UpdateSubmission.java

```diff
@@ -0,0 +1,130 @@
+package uk.gov.companieshouse.overseasentitiesapi.model.updateSubmission;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import uk.gov.companieshouse.overseasentitiesapi.model.dto.OverseasEntitySubmissionDto;
+import uk.gov.companieshouse.overseasentitiesapi.model.updateSubmission.changes.Change;
+
+import java.util.List;
+
+public class UpdateSubmission {
+    @JsonProperty("type")
+    private String type = "OE02";
+
+    @JsonProperty("userSubmission")
+    private OverseasEntitySubmissionDto userSubmission;
+
+    @JsonProperty("dueDiligence")
+    private DueDiligence dueDiligence;
+
```

Comment:
Need anOverseasEntitiesDueDiligence field !

Reply:
yes, see comment above.

Reply:
I've updated the ODP with the field mapping: https://companieshouse.atlassian.net/wiki/spaces/ROE2/pages/4144005155/ODP-ROE-Update-012-B+Change+list+Calculation#Due-Dilligence